kano_updater/scenarios.py

```diff
@@ -965,4 +977,10 @@ def beta_3_14_0_to_beta_3_14_1(self):
         pass
 
     def beta_3_14_1_to_beta_3_15_0(self):
-        pass
+        try:
+            import shutil
+            import os
+
+            shutil.copy(PIP_CONF, PYDIST_PATH)
```

Comment:
Why are we installing the .pth file in a post-update? Doesn't this mean there is a gap when those modules are not available? What happens if the user pulsl theplug just after the pre-update?

Reply:
The problem I have here is that `pre-update` is moving the `dist-packages` to the fallback directory and as a result the `pth` file that is installed through the debian `control` file. So, I decided to install the `pth` in a system directory and copy it `post-update`.

Do you have a better idea/recommendation for this?

Reply:
You can put the `.pth` file in `/usr/lib/python2.7/dist-packages` instead of in `/usr/local/lib`

Reply:
Thanks for the recommendation @Ealdwulf. Done and squashed.